ingestion/src/metadata/orm_profiler/profiler/handle_partition.py

```diff
@@ -132,7 +132,8 @@ def build_partition_predicate(partition_details: dict, col):
             f"{partition_details['partition_field']} in "
             f"({format_partition_values(partition_details['partition_values'], col_type)})"
         )
-
+    if partition_details["partition_field"] == "_PARTITIONDATE":
+        col_type = "DATE"
```

Comment:
_PARTITIONDATE col_type is sent as None
so in order to have that available in `YYYY-MM-DD` format and not `YYYY-MM-DD HH-MM-SS`, had to add col_type as DATE